### PR TITLE
UML diagrams for DG (Exit, Help & Unknown Commands)

### LINKED ISSUE #98: Add Exit, Help and Unknown Commands into DG
- No sequence diagrams needed for `Unknown` and `Help` commands as they are considered trivial
- Detailed sequence diagram for the `Exit` Command, depicting the flow from start in `FindOurSEP` to end of program.

### PR DESCRIPTION
**Pull request by ehz0ah**
Close #98 

Achieved all requirements stated in the tagged issue

### CODE REVIEW

**Review by paulktham — APPROVED** [review]
LGTM! Nice work!

File: docs/UML_Diagrams/ExitSequence.drawio.svg
Comment (Holy-An) [inline docs/UML_Diagrams/ExitSequence.drawio.svg:1]: The dotted line is protuding out of its lifeline for exit command

**Review by Holy-An — CHANGES_REQUESTED** [review]
Good work, just a minor change to fix

File: docs/UML_Diagrams/ExitSequence.drawio.svg
Comment (ehz0ah) [reply docs/UML_Diagrams/ExitSequence.drawio.svg:1]: Good catch!

**Review by Holy-An — APPROVED** [review]
lgtm!

**Review by Holy-An — APPROVED** [review]
LGTM

File: docs/UML_Diagrams/ExitSequence.drawio.svg
Comment (isaacsaw25) [inline docs/UML_Diagrams/ExitSequence.drawio.svg:1]: The return type 'boolean' to `FindOurSEP` in the loop frame shouldn't have the colon 😅

File: text-ui-test/EXPECTED.TXT [hunk 1]
```diff
@@ -9,6 +9,10 @@ Please choose 1 or 2 or exit :)
 Hi! Welcome to FindOurSEP! Enter 'help' for the list of commands.
 --------------------------------------------------------------------------------
 --------------------------------------------------------------------------------
+This file don't exist...Don't make me go through your folder for nothing leh :(.
```
Comment (isaacsaw25) [inline text-ui-test/EXPECTED.TXT:12]: Interesting use of Singlish here 😅

**Pull request merged**